backend/game/engine/player_queue.py

```
@@ -0,0 +1,45 @@
+import itertools
+
+
+class GamePlayers():
+
+    def __init__(self):
+        self.all_players = []
+        self.dead_players = []
+        self.alive_players = []
+        self.current_player = None
+
+    def _add_initial_player(self, player):
+        self.all_players.append(player)
+
+    def _remove_initial_player(self, player):
+        if player in self.all_players:
```

If it's the case that they're not in there, then it should throw an error.